docs/install.rst

```diff
@@ -191,6 +189,9 @@ via `setuptools`_ after cloning the `git`_ repository.
 
 Using :command:`easy_install` is no longer recommended.
```

Comment:
should also be removed

Reply:
Removed the left over recommendation against using easy_install, so that easy_install isn't mentioned at all anymore on the install page.

Also fixed punctuation and added the `pip --version` note for checking the pip version first.